tools/esdlcmd/esdlcmd_monitor.cpp

```diff
@@ -973,14 +973,27 @@ class EsdlMonitorCmd : public EsdlConvertCmd
 
         xform->setParameter("diffmode", "'Monitor'");
         xform->setParameter("diffaction", "'Create'");
+
+        xform->setParameter("platform", "'roxie'");
```

Comment:
Ok I guess I'll change ESP to lower after all.  Lower is the standard way of saying "don't use case".